hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/ScheduleServiceImpl.java

```diff
@@ -58,39 +52,25 @@ private Schema scheduleSchema() {
 
     private static final class ScheduleServiceSchema extends Schema {
         public ScheduleServiceSchema() {
-            super(ScheduleServiceImpl.CURRENT_VERSION);
+            super(CURRENT_VERSION);
```

Comment:
I've been using GENESIS_VERSION for all of these so far. I think the version anyway would be 0.42 or 0.43. But if you use genesis (SemanticVersion.DEFAULT), then it will always get used first, and our subsequent releases can be numbered.

Reply:
I'll make that change in the next PR.